src/index.html

```diff
@@ -1,17 +1,197 @@
+          How old are You?
+
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="box">
+          Full date of birth:
+
+          <input
+          type= "date"
+          id ="date"
+          name ="date"
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+
+          <input type="checkbox">
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="box">
+          E-mail:
+
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label>
+          Password:
+
+          <input
+            type="password"
+            id="password"
+            name="password"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <div class="box">
+          Do you love cats?
+
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            value="yes"
+            >
+            Yes
+
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="loveCats"
+            value="no"
+            >
+            No
+
+          </label>
+        </div>
+        <label class="box">
+          What is your favorite color?
+
+          <input
+          type="color"
+          name="color"
+          >
+        </label>
+        <label class="box">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            id="time"
+            name="time"
+          >
+        </label>
+        <label class="box">
+          What are your favorite brand of cars?
+
+          <select
+            id="level"
+            name="level"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+
+          <input
+            type="range"
+            id="range"
+            name="range"
+            min="0"
+            max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+
+        <label class="box">
+          Comments:
+
+          <textarea
+            placeholder="Tell us about yourself"
+            minlength="10"
+            maxlength="500"
+            required>
+          </textarea>
```

Comment:
```suggestion
          <textarea
            placeholder="Tell us about yourself"
            minlength="10"
            maxlength="500"
            required
        ></textarea>
```

it should fix extra spaces in textarea field